.github/workflows/pull_requests.yml

```diff
@@ -21,8 +21,8 @@ jobs:
         npm install
         npm run setheapsize
 
-    - name: Lint
-      run: npx grunt lint
+    - name: Format & Lint
+      run: (npm run style || npm run lint) && git diff --exit-code
```

Comment:
Modified pull_request workflow to include prettier formatting. These checks cannot be made individually in separate steps because it changes the files. Hence doing it together and checking for any file changes